pkgs/applications/networking/instant-messengers/telegram/kotatogram-desktop/default.nix

```diff
@@ -0,0 +1,134 @@
+{ mkDerivation, lib, fetchFromGitHub, fetchsvn, fetchpatch
+, pkgconfig, pythonPackages, cmake, ninja, gcc9, qtbase
+, qtimageformats, gtk3, libappindicator-gtk3, libnotify
+, xdg_utils, ffmpeg_4, openalSoft, lzma, lz4, zlib, minizip
+, openssl, libtgvoip, rlottie-tdesktop, range-v3
+}:
+
+with lib;
+
+mkDerivation rec {
+  pname = "kotatogram-desktop";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "kotatogram";
+    repo = "kotatogram-desktop";
+    rev = "k${version}";
+    sha256 = "156r229dpxnyaxxgp1dr4vz8jla5p7r0dydhgdgk2yx1p11z6m3w";
+    fetchSubmodules = true;
+  };
+
+  # Arch patches (svn export telegram-desktop/trunk)
+  archPatches = fetchsvn {
+    url = "svn://svn.archlinux.org/community/telegram-desktop/trunk";
+    # svn log svn://svn.archlinux.org/community/telegram-desktop/trunk
+    rev = "512849";
+    sha256 = "1hl7znvv6qr4cwpkj8wlplpa63i1lhk2iax7hb4l1s1a4mijx9ls";
+  };
+
+  animatedStickersPatch = fetchpatch {
+    url = "https://github.com/kotatogram/kotatogram-desktop/commit/a196b0aba723de85ee573594d9ad420412b6391a.patch";
+    sha256 = "1mzfvcgj42r05k27423hbkvgg0gjrm3xv9hvadfqaq1ys46bpqi0";
+  };
+
+  patches = [
+    ./tdesktop.patch
+    "${archPatches}/no-gtk2.patch"
+    ./Use-system-wide-font.patch
+    animatedStickersPatch
```

Comment:
This looks like something that can be optional. Add `animatedStickers` argument with default value set to `true` and then do `] ++ optional animatedStickers animatedStickersPatch;`

Reply:
animatedStickersPatch is a fix for animated stickers. In k1.1.1 (and tdesktop v1.9.1) there is a bug with flickering stickers

Reply:
Ah, okay.